drivers/ipvlan/ipvlan_joinleave.go

```diff
@@ -121,6 +121,7 @@ func (d *driver) Join(nid, eid string, sboxKey string, jinfo driverapi.JoinInfo,
 
 // Leave method is invoked when a Sandbox detaches from an endpoint.
 func (d *driver) Leave(nid, eid string) error {
+	defer osl.InitOSContext()()
```

Comment:
I thought about that.. but its a good practice to add it in all the driver API calls (in case some netns specific operation happens in the future in a function that is called from multiple levels under this function)